package.nls.zh-cn.json

```diff
@@ -1,75 +1,89 @@
 {
-    "bookmarks.activitybar.title": "书签",
```

Comment:
Could you please set the indent to 4 spaces as other files do this too.

And it can make changes to this file easier to review